R/lookups.R

```diff
@@ -164,3 +164,21 @@ datasource_lk <- list(
   # Country population projections by age group
   un_age_projections = "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/CSV_FILES/WPP2019_PopulationBySingleAgeSex_2020-2100.csv"
 )
+
+#' @title Function to get bounding boxes for maps by WHO region
+#' @description Used internally in viz_maps.R functions for maps
+#' @returns a vector of class bbox (from the sf package)
+#'  with the min and max xy coordinates of the bounding box
+#' @keywords internal
```

Comment:
Saw your comment RE: internal. That's fine with me, I think this is more documented than it was anyways